sdk/tests/utilities/id_generator.py

```diff
@@ -0,0 +1,62 @@
+import uuid
+
+
+class IdGenerator:
+
+    def __init__(self, default_scope=None):
+        """
+
+        Parameters
+        ----------
+        default_scope   scope to use for subsequent generate calls when do scope is provided
+        """
+        self.default_scope = default_scope if default_scope is not None else "sdk_example"
```

Comment:
This may be for my education, as I've seen this used before, but any reason why we don't just set the `default_scope` to 'sdk_example' in line 6 directly?

Reply:
nope, @jyaffe raised the same!